pipeline/03-evaluate.R

```diff
@@ -201,15 +191,6 @@ gen_agg_stats <- function(data, truth, estimate, bldg_sqft,
       .groups = "drop"
     ) %>%
     ungroup() %>%
-    # COD, PRD, and PRB all output to a list. We can unnest each list to get
-    # additional info for each stat (95% CI, sample count, etc)
-    tidyr::unnest_wider(cod) %>%
-    tidyr::unnest_wider(COD_CI, names_sep = "_") %>%
-    tidyr::unnest_wider(prd) %>%
-    tidyr::unnest_wider(PRD_CI, names_sep = "_") %>%
-    tidyr::unnest_wider(prb) %>%
-    tidyr::unnest_wider(PRB_CI, names_sep = "_") %>%
-    # Rename columns resulting from unnesting
```

Comment:
The unnesting code here is no longer needed since the bootstrap outputs are no longer output to a list in each column.